lib/cdo/http_cache.rb

```diff
@@ -93,6 +94,7 @@ class HttpCache
     basketball
     dance
     dance-2019
+    dance-ai-2023
```

Comment:
Is it possible to collapse these into a single pattern?

Reply:
Not easily, because these are inserted into the middle of a path string:

![image](https://github.com/code-dot-org/code-dot-org/assets/7014619/c8c747f7-1844-4d16-be65-9bb0508a7f61)

Reply:
It might be possible to do something like `dance*` ([pathPattern docs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-values-specify.html?icmpid=docs_cf_help_panel#DownloadDistValuesPathPattern))

Reply:
#54980 for your consideration.